sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/ConsumerSpEL.java

```diff
@@ -88,4 +106,38 @@ public long getRecordTimestamp(ConsumerRecord<byte[], byte[]> rawRecord) {
     }
     return timestamp;
   }
+
+  @SuppressWarnings("unchecked")
+  public long offsetForTime(Consumer<?, ?> consumer, TopicPartition topicPartition, Instant time) {
```

Comment:
A short JavaDoc is useful. Could be just one or two sentences, clarifying it throws if offset could not be fetched for any reason.